Modules/feed/engine/PHPFina.php

```diff
@@ -336,10 +336,8 @@ public function get_data($name,$start,$end,$interval,$skipmissing,$limitinterval
             }
             
             if ($value!==null || $skipmissing===0) {
-                // Re-enable see https://openenergymonitor.org/emon/node/11260
-                // if ($time>=$start && $time<$end) {
```

Comment:
It might be useful to leave this comment in for now, I nee to look at this again to ensure the behaviour is correct